apps/src/templates/curriculumCatalog/ExpandedCurriculumCatalogCard.jsx

```diff
@@ -15,33 +23,167 @@ const ExpandedCurriculumCatalogCard = ({
+                    href={professionalLearningProgram}
+                    icon={
+                      <FontAwesome
+                        icon="arrow-up-right-from-square"
+                        className="fa-solid"
+                      />
+                    }
+                  />
+                  <hr className={style.horizontalDivider} />
+                  <TextLink
+                    text={i18n.selfPacedPl()}
+                    href={selfPacedPlCourseOfferingPath}
+                    icon={
+                      <FontAwesome
+                        icon="arrow-up-right-from-square"
+                        className="fa-solid"
+                      />
+                    }
+                  />
+                </div>
+              </div>
+            </div>
+            <hr className={style.horizontalDivider} />
+            <div className={style.compatibilityContainer}>
+              {Object.keys(devices).map(device => (
+                <div className={style.iconWithDescription}>
+                  <FontAwesome
+                    icon={iconData[devices[device]].icon}
+                    className={`fa-solid ${iconData[devices[device]].color}`}
+                  />
+                  <BodyTwoText>
+                    {device !== 'no_device'
+                      ? translatedCourseOfferingDeviceTypes[device]
+                          .charAt(0)
+                          .toUpperCase() +
+                        translatedCourseOfferingDeviceTypes[device].slice(1)
+                      : i18n.offline()}
+                  </BodyTwoText>
+                </div>
+              ))}
+            </div>
+            <hr className={style.horizontalDivider} />
+            <div className={style.buttonsContainer}>
+              <Button
+                __useDeprecatedTag
+                color={Button.ButtonColor.neutralDark}
+                type="button"
+                href={pathToCourse}
+                aria-label={i18n.quickViewDescription({
+                  course_name: courseDisplayName,
+                })}
+                text={i18n.seeCurriculumDetails()}
+                style={{flex: 1}}
+              />
+              <Button
+                color={Button.ButtonColor.brandSecondaryDefault}
+                type="button"
+                onClick={assignButtonOnClick}
+                aria-label={assignButtonDescription}
+                text={i18n.assignToClassSections()}
+                style={{flex: 1}}
+              />
+            </div>
           </div>
-          <div className={style.iconWithDescription}>
-            <FontAwesome icon="book" className="fa-solid" />
-            <BodyTwoText>{subjectsAndTopics.join(', ')}</BodyTwoText>
+          <div className={style.relatedCurriculaContainer}>
+            <div className={style.closeButton}>
+              <a onClick={onClose}>
+                <FontAwesome icon="xmark" className="fa-solid" />
+              </a>
```

Comment:
You should generally only use `<a>` tags when using an `href` prop, but if you're using an `onClick` please use `Button` tags as it allows people to tab navigate to them